runner/goroutine/goroutine.go

```diff
@@ -42,6 +42,11 @@ func (g *Runner) Stop() error {
 	return nil
 }
 
+// IsReady will always return nil for goroutine runner
+func (g *Runner) IsReady() error {
+	return nil
```

Comment:
I think you can check if the chan is not closed

Reply:
Checking that a channel is not closed is not straightforward, and I'm not sure that there is a demand for the method `IsReady` for goroutine runner, so I won't implement it in this MR